edi_oca/models/edi_exchange_record.py

```diff
@@ -110,11 +110,11 @@ class EDIExchangeRecord(models.Model):
         ),
     ]
 
-    @api.depends("type_id.code", "model", "res_id")
-    def _compute_name(self):
+    @api.depends("model", "res_id")
```

Comment:
this depends looks wrong?
```python
@api.depends("record", "record.display_name")
```

Reply:
is not :) `record` is a property computed from res_id and model ;)